submissions/OlStaniev/html-css-popup/css/style.scss

```diff
@@ -0,0 +1,209 @@
+/*Обнуление*/
+* {
+    padding: 0;
+    margin: 0;
+    border: 0;
+}
+
+*,
+*:before,
+*:after {
+    -moz-box-sizing: border-box;
+    -webkit-box-sizing: border-box;
+    box-sizing: border-box;
+}
+
+:focus,
+:active {
+    outline: none;
+}
+
+a:focus,
+a:active {
+    outline: none;
+}
+
+nav,
+footer,
+header,
+aside {
+    display: block;
+}
+
+html,
+body {
+    height: 100%;
+    width: 100%;
+    font-size: 100%;
+    line-height: 1;
+    font-size: 14px;
+    -ms-text-size-adjust: 100%;
+    -moz-text-size-adjust: 100%;
+    -webkit-text-size-adjust: 100%;
+}
+
+input,
+button,
+textarea {
+    font-family: inherit;
+}
+
+input::-ms-clear {
+    display: none;
+}
+
+button {
+    cursor: pointer;
+}
+
+button::-moz-focus-inner {
+    padding: 0;
+    border: 0;
+}
+
+a,
+a:visited {
+    text-decoration: none;
+}
+
+a:hover {
+    text-decoration: none;
+}
+
+ul li {
+    list-style: none;
+
+}
+
+img {
+    vertical-align: top;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+    font-size: inherit;
+    font-weight: 400;
+}
+
+/*--------------------*/
+* {
+    font-family: 'Indie Flower', cursive;
+}
+
+body {
+    position: relative;
+}
+
+.header {
+    background-color: black;
+    width: 100%;
+    height: 20vh;
+    color: rgb(250, 3, 106);
+    font-size: 40px;
+
+    &__wrapper {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        padding: 5rem;
+
+        &_logo {
+            font-size: 70px;
+            font-weight: bold;
+        }
+    }
+
+}
+
+.main {
+    width: 100%;
+    margin: 0 auto;
+    position: relative;
+
+
```

Comment:
Two empty lines. It is fine to leave an empty line between two blocks, but be consistant, you have only one line between other blocks.